src/js/components/Button/Button.js

```diff
@@ -169,6 +171,20 @@ const Button = forwardRef(
       );
     }
 
+    const sendAnalytics = useAnalytics();
+
+    const onClick = useCallback(
+      (event) => {
+        sendAnalytics({
+          type: 'buttonClick',
+          element: event.target,
+          data: event,
```

Comment:
Just confirming that "event" wouldn't be a more appropriate key name for the event?

Reply:
probably true at this point. I originally settled on `data` because there are cases where it's not tied to real event.